scripts-available/CDB_Groups_API.sql

```diff
@@ -0,0 +1,57 @@
+-- Requires configuration parameter. Example: SELECT cartodb.CDB_Conf_SetConf('groups_api', '{ "host": "127.0.0.1", "port": 3000, "timeout": 10, "username": "superadmin", "password": "monkey" }');
+
+-- Sends the create group request
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Group_CreateGroup_API(database_name text, group_name text, group_role text)
+    RETURNS VOID AS
+$$
+    import httplib
+    import base64
+    import string
+    import json
+
+    try:
+      conf = plpy.execute("SELECT cartodb.CDB_Conf_GetConf('groups_api') conf")[0]['conf']
```

Comment:
I understand you use json because in this way you avoid doing 3 requests. I'd prefer to have a plain table with key value being text or either force the field to be valid json (personal preference)